src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java

```diff
@@ -200,11 +199,10 @@ public Metadata getModMetadata() {
     }
 
     class WhiteLine {
-
         private double xPercent, yPercent;
         private int width;
 
-        public WhiteLine(double xPercent, double yPercent, int width) {
+        protected WhiteLine(double xPercent, double yPercent, int width) {
```

Comment:
Why is this necessary?

Reply:
Sorry, IntelliJ has a mind of its own. Removed.